static/app/utils/profiling/colors/utils.tsx

```diff
@@ -1,24 +1,27 @@
+import {SpanChart, SpanChartNode} from 'sentry/utils/profiling/spanChart';
+
 import {ColorChannels, FlamegraphTheme, LCH} from '../flamegraph/flamegraphTheme';
 import {FlamegraphFrame} from '../flamegraphFrame';
 
-const uniqueBy = <T,>(arr: ReadonlyArray<T>, predicate: (t: T) => unknown): Array<T> => {
+function uniqueBy<T>(arr: ReadonlyArray<T>, predicate: (t: T) => unknown): Array<T> {
```

Comment:
why not just use `lodash/uniqBy`

Reply:
I think this copied over from Specto when we werent using lodash, idk if we want to change this at some point, tbh it's a very simple implementation and not re-exported